includes/gateway/class-omise-payment-promptpay.php

```diff
@@ -146,6 +147,16 @@ public function display_qrcode( $order, $context = 'view' ) {
 				<div id="omise-offline-payment-timeout" style="margin-top: 2em; display: none;">
 					<p><button id="omise-offline-payment-refresh-status">refresh status</button></p>
 				</div>
+
+				<div class="qr-alert-box">
+					<p><strong>To make payment:</strong></p>
+					<ol class="align-left">
+						<li>Download the QR code or open your preferred bank app to scan it</li>
+						<li>Check that the payment details are correct</li>
+						<li>Import the QR code image into your bank app or scan the QR code with your bank app to pay.</li>
```

Comment:
```suggestion
						<li>Import the QR code image into your bank app or scan the QR code with your bank app to pay</li>
```

Reply:
For consistency. This is the only bullet point ending in a period.

Reply:
Thanks. Resolved.